straticounter.m

```diff
@@ -1,123 +1,183 @@
-function straticounter(sett_icecore)
-clc; close all;
-releasedate = '30-04-2015';
-
-%% StratiCounter: A layer counting algorithm
-% Developed by Mai Winstrup, mai@gfy.ku.dk
-
-% The algorithm is based on the principles of statistical inference of 
-% hidden states in semi-Markov processes. States, and their associated 
-% confidence intervals, are inferred by the Forward-Backward algorithm. 
-% The EM (Expectation-Maximization) algorithm is used to find the optimal 
-% set of layer parameters for each data batch. Confidence intervals do not 
+function straticounter(varargin)
+%% STRATICOUNTER A Layer counting algorithm
```

Comment:
I removed clc and release date from here because it will allow MATLAB to show a basic help screen for your function if you push f1. You will find them after the big comment block